packages/ui-extensions-sdk/src/config/config.ts

```diff
@@ -0,0 +1,14 @@
+import { RequestType } from '../constants';
+import { Config, ContextClient, LoggerClient, RequestClient } from '../types';
+
+export class DDConfigClient {
+    private readonly client: ContextClient & LoggerClient & RequestClient;
+
+    constructor(client: ContextClient & LoggerClient & RequestClient) {
+        this.client = client;
+    }
+
+    async getConfig(): Promise<Config> {
+        return this.client.request(RequestType.GET_CONFIG);
+    }
```

Comment:
I talked this over with @joneshf-dd — it feels best to implement this as a request since developers would likely want to request config values from `web-ui`. For full context, I briefly thought about packaging up config data in the `context`, but this would likely require better segmentation of the `context` object itself and dissociation between `framepost`'s `context` and the SDK Client's `context`.

Reply:
There's no doubt that the way you've done it is the simpler way, and I think it's right for now.

I see us almost certainly doing that work to dissociate framepost's `context` and the SDK's `context` anyways, at some point, in order to support a more rich data-model of information getting sent to the SDK. Hypothetically this could look something like:

```
{
  context: ...,
  orgConfig: ...,
  userConfig:..,
  state
}
```

but I don't know exactly what that will look like. We may even expose this data-structure directly from some hook like `client.getData()` or whatever. I talked with hardy about this and we agreed it's a good idea not to rush in to that before designing it carefully.

If I'm right that this is where we're heading, we'll have to go through a deprecation process of support for that request type down the line when we do ^ work, which will be very annoying, but at least we get to do that work after having defined the data-model more carefully.

Reply:
One thought though: 

What happens when if we end up doing arbitrary user-config later? Should we maybe name this `getOrgConfig()`? 

One general feedback about this feature cross-stack is that the naming feels inconsistent. We call it 'user settings' in the dev platform 'settings' in the installation modal, and 'config' here?

Reply:
I like the idea of renaming the method to `getOrgConfig`, and accordingly the request type, etc.

The naming is inconsistent across the stack, but I think each of the interfaces you mention carry slightly different context:
* In the Developer Platform, developers see "User Settings" since it's ultimately the user that will end up configuring these settings (I'm open to calling it "Org Settings", but I'm curious to hear Maureen's thoughts on it — if we decide Org Settings is better, then we can probably rename both the user-facing and developer-facing views the same)
* In my head, I view "configurations" as configured settings, whereas settings kind of just represents the universe of what can be toggled, so I'm okay with leaving it as "-config"

Reply:
Just recapping our offline conversation here — 

We're going with "Org Configuration" anytime developers are defining what's configurable and "Org Settings" anytime users are configuring individual settings.